packages/back-end/src/services/organizations.ts

```diff
@@ -247,6 +247,11 @@ export async function acceptInvite(key: string, userId: string) {
     throw new Error("Invalid key");
   }
 
+  // If memeber is already in the org, skip
```

Comment:
This is the fix for the duplicate entries in org.members. Now, we're checking to see if the userId that is trying to accept the invite is already in the org.members array, and if so, we're skipping the rest of the logic and later throwing an error.

Reply:
Why not just throw the error here?

Reply:
Updated.